src/Ztm.WebApi/InvalidCallbackUrlException.cs

```diff
@@ -0,0 +1,24 @@
+using System.Net;
+
+namespace Ztm.WebApi
+{
+    public sealed class InvalidCallbackUrlException : HttpResponseException
```

Comment:
We need to update this once we update `HttpResponseException`.

Reply:
done